website/tsconfig.json

```diff
@@ -1,11 +1,13 @@
 {
   "extends": "../tsconfig.json",
   "compilerOptions": {
+    "composite": true,
     "plugins": [{ "name": "next" }],
     "baseUrl": ".",
-    "jsx": "preserve",
+    "jsx": "react-jsx",
     "incremental": true,
     "isolatedModules": true,
+    "outDir": "./dist/",
     "paths": {
       "@/*": ["./*"]
     }
```

Comment:
This `include` directive below is pretty suspect. I updated it on purpose in [4325](https://github.com/ariakit/ariakit/pull/4325/files#diff-60e40184bc693f11abb73041e90164447d62590ab4d3685f5b6e44667c9261a4R14) to avoid unintentional file misses.

Reply:
Might be, just trying to keep the PR as minimal as possible. Good candidate for a separate PR I think!